client/luigi-client-wc-docu-mixin.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * <!-- label-success: Web Component API -->
```

Comment:
I don’t see syntax highlighting but that’s matching the rest of the code blocks. It is longer than most other examples but unless you could omit some declarations that are made in other parts of the guide, it is also fine. On capire the site generator has more plugins enabled which would be nice like highlighting only the relevant lines of code (for example: [link](https://cap.cloud.sap/docs/guides/using-services#mock-associations)) .